test_results/rspec.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
pretty sure you didn't mean to check this in

Reply:
Yeah I did an extra git add .
I'll remove that